avocado/utils/pci.py

```diff
@@ -133,7 +133,7 @@ def get_slot_from_sysfs(pci_address):
 
     :return: slot of pci address from sysfs.
     """
-    if not os.path.isdir('/sys/bus/pci/devices/%s' % pci_address):
+    if not os.path.isdir('/sys/bus/pci/devices/%s/devspec' % pci_address):
```

Comment:
`/sys/bus/pci/devices/%s/devspec` is not a directory. It should be `os.path.isfile()`